Mathlib/Probability/Cdf.lean

```diff
@@ -0,0 +1,88 @@
+/-
+Copyright (c) 2023 Rémy Degenne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémy Degenne
+-/
+import Mathlib.Probability.Kernel.CondCdf
+
+/-!
+# Cumulative distribution function of a real probability measure
+
+The cumulative distribution function (cdf) of a probability measure over `ℝ` is a monotone, right
+continuous function with limit 0 at -∞ and 1 at +∞, such that `cdf μ x = μ (Iic x)` for all `x : ℝ`.
+Two probability measures are equal if and only if they have the same cdf.
+
+## Main definitions
+
+* `ProbabilityTheory.cdf μ`: cumulative distribution function of `μ : Measure ℝ`, defined as the
+  conditional cdf (`ProbabilityTheory.condCdf`) of the product measure
+  `(Measure.dirac Unit.unit).prod μ` evaluated at `Unit.unit`.
+
+## Main statements
+
+* `ProbabilityTheory.ofReal_cdf`: for a probability measure `μ` and `x : ℝ`,
+  `ENNReal.ofReal (cdf μ x) = μ (Iic x)`.
+* `MeasureTheory.Measure.ext_of_cdf`: two probability measures are equal if and only if they have
+  the same cdf.
+
+## TODO
+
+The definition could be extended to a finite measure by rescaling `condCdf`, but it would be nice
+to have more structure on Stieltjes functions first. Right now, if `f` is a Stieltjes function,
+`2 • f` makes no sense. We could define Stieltjes functions as a submodule.
+
+The definition could be extended to `ℝⁿ`, either by extending the definition of `condCdf`, or by
+using another construction here.
+-/
+
+open MeasureTheory Set Filter
+
+open scoped Topology
+
+namespace ProbabilityTheory
+
+/-- Cumulative distribution function of a real measure. -/
```

Comment:
Can you add a warning that it only really makes sense for probability measures, and add a pointer to `ofReal_cdf` (which is maybe the more usual definition of the cdf)?